`Button`: Add isInline argument (HDS-2681) (#1854)
hashicorp/design-system: website/docs/components/button/partials/code/how-to-use.md

<Hds::Button @text="Full width button" @isFullWidth={{true}} />
```

> ### Layout

Author: Pushed updates

<Hds::Button @text="Full width button" @isFullWidth={{true}} />
```

### Layout

> To change the default `block` layout to `inline`, set `@isLine` to `true`.
> 
> In contexts where the Button needs to be <em>inline</em>, to inherit the alignment from a parent, you can use the `@isInline` argument

Reviewer: I see the two paragraph as a bit of duplication. What do you think of merging them in a single concept?

```suggestion
To change the default `block` layout to `inline`, set `@isLine` to `true`. This can be useful in contexts where the Button needs to be <em>inline</em>, for example to inherit the alignment from a parent.
```

> <!-- #### Default block layout
> 
> ```handlebars
> <Hds::Button @text="default block layout" />
> <Hds::Button @text="default block layout" />
> ``` -->
> 
> <!-- #### Inline layout -->
> 

Reviewer: We (generally) don't leave commented code in the documentation (it may cause issues with the parser).

```suggestion
```

Author: Sorry that was an oversite. I meant to delete it
